### ISSUE TITLE
Problem with state of SpanningSheet widget

### ISSUE THREAD

**Issue by II11II** [AdamJonsson/snapping_sheet#2]
My GoogleMap Widget which is wrapped with SpanningSheet Widget is not visible after swiping down the SpanningSheet widget
[https://github.com/II11II/weather](url) <-here is full code
[https://t.me/dartlang_group/62861](url)

**Comment by II11II** [comment]
Unhandled Exception: 'package:flutter/src/services/platform_views.dart': Failed assertion: line 548 pos 12: '!size.isEmpty': is not true.

**Comment by AdamJonsson** [comment]
Hi, thanks for creating an issue. The problem seems to be that the map widget can not be resized to zero height. As the sheetBelow child do this when collapsed, this creates a problem.

I have had similar problems in other project and I found that the best solution is to wrap the GoogleMaps widget with a Stack and Positioned. This prevents the map to shrink to zero height when the sheet is collapsed.

I made a fork of your project and created pull request that can be found here that fixes this problem: https://github.com/II11II/weather/pull/1/files

Let me know if it works for you 😄
